eng/source-build-patches/0001-Remove-Yarn-dependency-not-used-in-source-build.patch

```diff
@@ -25,19 +25,19 @@ index 696d23402b..6086dd0843 100644
      <NormalizedPackageId>$(PackageId.Replace('@','').Replace('/','-'))</NormalizedPackageId>
      <PackageFileName>$(NormalizedPackageId)-$(PackageVersion).tgz</PackageFileName>
 diff --git a/global.json b/global.json
-index 501ba161bf..ee9dcaf4fa 100644
+index 5a2b8a613d..fa6d0c7954 100644
 --- a/global.json
 +++ b/global.json
 @@ -29,7 +29,6 @@
      "xcopy-msbuild": "16.5.0-alpha"
    },
    "msbuild-sdks": {
 -    "Yarn.MSBuild": "1.22.10",
-    "Microsoft.DotNet.Arcade.Sdk": "6.0.0-beta.21167.3",
-    "Microsoft.DotNet.Helix.Sdk": "6.0.0-beta.21167.3"
+     "Microsoft.DotNet.Arcade.Sdk": "6.0.0-beta.21264.2",
```

Comment:
We need a more sustainable way of patching/modifying global.json. With the current method, we need to update this patch every time arcade is updated.

Reply:
The intent here is to eliminate the patches (https://github.com/dotnet/aspnetcore/issues/31446).  They should be integrated into the product.  Without this, you are right they are not sustainable which is the driving for behind the ArPow effort.

Reply:
Yea, what I meant here is I'm not aware of "proper" ways to modify global.json files depending on whether we're building for source build. How are other repos handling these modifications? It would surprise me if we are the only ones who need changes in global.json.

Reply:
Added a script for modifying the global.json